warehouse/clickhouse/cluster/clickhouse01/config.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0"?>
```

Comment:
Are those files used only for testing?

If yes a good practise is to put them under `testdata/` folder, next to the `_test.go` file.

Reply:
Moved test config files to testdata/

Reply:
Done.